sf_api.js

```diff
@@ -31,4 +31,12 @@ updateReconciliationText = function(id, attributes) {
   return axios.post(`reconciliations/${id}`, attributes, axiosConfig())
 }
 
-module.exports = { fetchReconciliationTexts, updateReconciliationText, findReconciliationText }
+createTestRun = function(attributes) {
```

Comment:
Missing `const` and possible usage of arrow function with implicit return (since it's a one liner)

Reply:
@ihcene I'd rather keep all functions consistent then some arrow functions and some not

Reply:
You are right